[2024-05-02T20:19:07.032+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: diabetes_prediction.fetch_dataset manual__2024-05-02T20:19:02.585507+00:00 [queued]>
[2024-05-02T20:19:07.053+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: diabetes_prediction.fetch_dataset manual__2024-05-02T20:19:02.585507+00:00 [queued]>
[2024-05-02T20:19:07.053+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2024-05-02T20:19:07.080+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): fetch_dataset> on 2024-05-02 20:19:02.585507+00:00
[2024-05-02T20:19:07.095+0000] {standard_task_runner.py:57} INFO - Started process 76 to run task
[2024-05-02T20:19:07.102+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'diabetes_prediction', 'fetch_dataset', 'manual__2024-05-02T20:19:02.585507+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpya113bw9']
[2024-05-02T20:19:07.106+0000] {standard_task_runner.py:85} INFO - Job 137: Subtask fetch_dataset
[2024-05-02T20:19:07.199+0000] {task_command.py:410} INFO - Running <TaskInstance: diabetes_prediction.fetch_dataset manual__2024-05-02T20:19:02.585507+00:00 [running]> on host 8baaf4bc95aa
[2024-05-02T20:19:07.408+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='diabetes_prediction' AIRFLOW_CTX_TASK_ID='fetch_dataset' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T20:19:02.585507+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-02T20:19:02.585507+00:00'
[2024-05-02T20:19:09.011+0000] {warnings.py:110} WARNING - /opt/***/dags/dag1.py:14: DtypeWarning: Columns (10) have mixed types.Specify dtype option on import or set low_memory=False.
  data = fetch_ucirepo(id=296)

[2024-05-02T20:19:09.153+0000] {warnings.py:110} WARNING - /opt/***/dags/dag1.py:18: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df['readmitted'] = y.readmitted

[2024-05-02T20:19:10.059+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-05-02T20:19:10.078+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=diabetes_prediction, task_id=fetch_dataset, execution_date=20240502T201902, start_date=20240502T201907, end_date=20240502T201910
[2024-05-02T20:19:10.140+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2024-05-02T20:19:10.204+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
